CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Task annotations importing via chunk uploads (<https://github.com/openvinotoolkit/cvat/pull/4327>)
 - Advanced filtration and sorting for a list of tasks/projects/cloudstorages (<https://github.com/openvinotoolkit/cvat/pull/4403>)
+- Support paginated list for job commits (<https://github.com/openvinotoolkit/cvat/issues/4476>)
```

Comment:
```suggestion
- Support paginated list for job commits (<https://github.com/openvinotoolkit/cvat/issues/4476>)
```
```suggestion
- Support paginated list for job commits (<https://github.com/openvinotoolkit/cvat/pull/4482>)
```

Reply:
@Marishka17 , Done. thanks for your comment.